src/models/RevAiApiError.ts

```diff
@@ -3,14 +3,14 @@ import { AxiosError } from 'axios';
 export class RevAiApiError {
     statusCode: number;
     title: string;
-    detail?: string;
+    error?: string;
     type?: string;
 
     constructor(e: AxiosError) {
         if (e.response) {
             this.statusCode = e.response.status;
             this.title = e.response.data.title || '';
-            this.detail = e.response.data.detail || '';
+            this.error = e.response.data.error || '';
             this.type = e.response.data.type || '';
```

Comment:
I'm also not sure why we're trying to filter out what's important here. Why not just show the user just response.status and response.data?

Reply:
It makes it difficult to keep updated if the specific field changes. I'm guessing that's what happened here and the sdks never got updated and we switched to using "error" instead of "detail".

Reply:
Went ahead and applied this change